src/Search/Microsoft.Azure.Search/Customizations/Indexers/Models/IndexingParametersExtensions.cs

```diff
@@ -13,24 +13,6 @@ namespace Microsoft.Azure.Search.Models
     /// </summary>
     public static class IndexingParametersExtensions
     {
-        private const string ParsingModeKey = "parsingMode";
-
-        /// <summary>
-        /// Specifies that the indexer will index only the storage metadata and completely skip the document extraction process. This is useful when
-        /// you don't need the document content, nor do you need any of the content type-specific metadata properties.
-        /// See <see href="https://docs.microsoft.com/azure/search/search-howto-indexing-azure-blob-storage" /> for details.
-        /// </summary>
-        /// <param name="parameters">IndexingParameters to configure.</param>
-        /// <remarks>
-        /// This option only applies to indexers that index Azure Blob Storage.
-        /// </remarks>
-        /// <returns>The IndexingParameters instance.</returns>
-        [Obsolete("This method is obsolete. Please use SetBlobExtractionMode(BlobExtractionMode.StorageMetadata).")]
```

Comment:
@chaosrealm I didn't think the stakes were very high since removing these doesn't prevent any scenario from actually working. But it is true that they haven't been obsolete for very long. :-)  I'll put them back in.